db-init/src/main/resources/database/migrations/V3.0__Create_Event_Table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS audit_event (
+    id uuid DEFAULT uuid_generate_v4 () NOT NULL,
+    event_id VARCHAR NOT NULL,
+    route_id VARCHAR,
+    payload_type VARCHAR NOT NULL,
+    message VARCHAR,
+    throwable TEXT,
+    details TEXT,
```

Comment:
How can we ensure that no PII/PHI is stored in the DB?

Reply:
eventId, routeId, palyload_type do not contain PII. The message and details is under the control of the developer. This leaves the possibility that some PII makes into the stacktrace somehow, but in this case it would also go to the log files, which is a bigger problem.

Reply:
I'm referring to the `details` field. What data goes into that and how can we ensure that a developer does not put PII in it? Perhaps just adding a reminder in code comments?